frontend/src/components/initialPage/InitialHeader.jsx

```diff
@@ -0,0 +1,33 @@
+import { Link } from "react-router-dom"
+
+const NavLinks = () => {
+    return(
+      <>
+        <Link to="/SignUp">Sign Up</Link>
+      </>
+    );
+  };
+
+
+
+const InitialHeader = () => {
+    return (
+      <div >
+        <header className='fixed z-[1] top-0 w-full flex flex-wrap items-center justify-between bg-background p-[2em] font-sans font-bold backdrop-blur-[100px] bg-slate-900 text-white'>
+                <div className='m-3 items-center'>
+                  <h2 className='font-bebas text-xl text-3xl'>Event finder</h2>
+                </div>
+
+                <div className='content-center pl-[2rem]'>
+                  <div className="hidden justify-end lg:flex" id='component'>
+                      <NavLinks />
```

Comment:
Redundant code here with the creation of InitialHeader here. This is largely similar to Header, except it is for when user not logged in. You can achieve this by use a state to tell whether user has logged in or not. If they have, load loggedInNavLinks; if they have not load LoggedOutNavLinks.

Reply:
To do (need to solve user auth first)

Reply:
Done